# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../recode/gcc43.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur recode-3.6-orig/src/recodext.h recode-3.6/src/recodext.h
--- recode-3.6-orig/src/recodext.h	2010-08-30 11:40:16.000000000 +0200
+++ recode-3.6/src/recodext.h	2010-08-30 11:41:23.000000000 +0200
@@ -218,7 +218,7 @@
     enum recode_symbol_type type : 3;
 
     /* Non zero if this one should be ignored.  */
-    bool ignore : 2;
+    bool ignore : 1;
   };
 
 struct recode_surface_list
